drivers/spi/Kconfig.nrfx

```diff
@@ -14,6 +14,20 @@ menuconfig SPI_NRFX
 
 if SPI_NRFX
 
+if SOC_NRF52832
+
+config SOC_NRF52832_PAN_58_OVERRIDE
+	bool "Allow enabling the SPIM driver despite PAN 58"
+	help
+		Allow enabling the nRF SPI Master with EasyDMA, despite
```

Comment:
Use the same indentation as it is used for other options in this file, please.

Reply:
Apologies - the mixed tab/spaces leading to differing indentation escaped my notice. 

Fixed.